multimodal/src/autogluon/multimodal/utils/inference.py

```diff
@@ -59,6 +74,25 @@ def extract_from_output(outputs: List[Dict], ret_type: str, as_ndarray: Optional
         return [ele[TEXT] for ele in outputs]  # single image
     elif ret_type == SCORE:
         return [ele[SCORE] for ele in outputs]
+    elif ret_type == NER:
```

Comment:
It seems that `NER` is used as both the model key and return type. How about using different names for them?

Reply:
what about NER_RET?

Reply:
`NER_RET ` is fine. It would be better if being more specific about return contents.

Reply:
It is done in the latest commit.